index.html

```diff
@@ -247,6 +257,58 @@ <h3>Skills</h3>
                   </div>
                 </div>
               </section>
-             </main>
-            </body>
-          </html>
+
+              <!--Form section-->
+    <section id="form-section">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form
+        action="#"
```

Comment:
- [ ] Kindly use the form service provided in the requirement for the action, so the action will be something like `action='https://formspree.io/f/{form_id}'`, you can find the form service in this [link](https://formspree.io/html/)